src/monster.cpp

```diff
@@ -50,7 +50,7 @@ Monster::Monster(MonsterType* mType) :
 	defaultOutfit = mType->info.outfit;
 	currentOutfit = mType->info.outfit;
 	skull = mType->info.skull;
-	health = std::min(mType->info.health, mType->info.healthMax);
+	health = mType->info.health
```

Comment:
missing ;

Reply:
`std::min` is no longer needed because the MonsterType on loading fixes the healthNow overflow, so there will never be `mType->info.health` greater than `mType->info.healthMax`
[monsters.cpp](https://github.com/otland/forgottenserver/pull/3116/commits/8160012a428efad21427528e309f7dd280e1307e#diff-721c64f80744c3e002360fff5827e2c8R883) and [register_monster_type.lua](https://github.com/otland/forgottenserver/pull/3116/files#diff-77b54c98309c519e7f79ba6b583c1826R39)

Reply:
He fixed the missing ; in this commit https://github.com/otland/forgottenserver/pull/3116/commits/04558bcd14b2cbfd876abdd6d96166a4d8b9b921